src/pymor/core/pipeline.py

```diff
@@ -130,7 +130,7 @@ def _run_prefect(self, data, rule_spec):
         logger.debug("Dynamically creating workflow with DaskTaskRunner...")
         cmor_name = rule_spec.get("cmor_name")
         rule_name = rule_spec.get("name", cmor_name)
-        if self._cluster is None:
+        if not hasattr(self, "_cluster") or self._cluster is None:
```

Comment:
This is clearer:
```suggestion
        if getattr(self, "_cluster", None) is None:
```

However, I'd rather check for something like if `self._cluster` is an instance of a dask Cluster class, but I am unsure exactly how to check that. Something like:

```python
cluster = getattr(self, "_cluster", None)
cluster_type = ??? # <-- No idea what goes here
if not isinstance(cluster, cluster_type):
    ...
```